### ISSUE TITLE
Missing terrain in Brazillian airport Curitiba (SBCT)

### ISSUE THREAD

**Issue by kubilus1** [kubilus1/autoortho#239]
### Discussed in https://github.com/kubilus1/autoortho/discussions/200

<div type='discussions-op-text'>

<sup>Originally posted by **rquintelarj** June  5, 2023</sup>
Hello!

Using AutoOrtho, nearby and at Curitiba airport (SBCT - XP11 and 12), the scenery disappears, becoming like the image.

Does anyone know how to fix?

![X-Plane Screenshot 2023 06 05 - 15 19 11 40](https://github.com/kubilus1/autoortho/assets/135556412/5279cd17-d7cc-4494-8081-b7e824f35be3)
</div>

**Comment by d41k4n** [comment]
This is another OSM coastline error which caused Ortho4XP to generate a corrupted DSF for tile -26-050. 

It's now fixed in OSM with my [changeset #138039362](https://www.openstreetmap.org/changeset/138039362). Pending the next batch-rebuild of the DSF's by @kubilus1 this will hopefully be included in an upcoming AutoOrtho release.

Technical background: The issue was caused by 4 lone islets wrongly tagged with `natural=coastline` and which should have been part of the existing multipolygon relation ["Represa do Rio Verde"](https://www.openstreetmap.org/relation/574722) to begin with...

I added them as inner members of that multipolygon and removed the coastline tag for each of them:

![Clipboard01](https://github.com/kubilus1/autoortho/assets/6691087/34efe342-7a23-4bee-b148-c278d571803c)

e.g.

![Clipboard02](https://github.com/kubilus1/autoortho/assets/6691087/57463582-ba6c-44ef-967b-7f2d3968e6f6)

After deleting cached OSM data in Ortho4XP I could successfully validate the mesh was created correctly for tile -26-050.
